### PR TITLE
Modified visited: state for link in footer to white colour

### PR DESCRIPTION
**Pull request by boonier**

### CODE REVIEW

**Comment by boonier** [comment]
@roobottom Jon I've updated this could you take a look please? Does this need to be replicated in the ff_container-dashboard-section?

File: blocks/melody/ff_container/ff_container-landing-section/ff_container-landing-section.less [hunk 1]
```diff
@@ -26,7 +26,8 @@
     &__more-link {
         #ff_mix-util-prose > .link-no-decoration();
```
Comment (dkhuntrods) [inline blocks/melody/ff_container/ff_container-landing-section/ff_container-landing-section.less]: @boonier You can pass colour vars in to the link-no-decoration mixin; you shouldn't need to add the states here

File: blocks/melody/ff_container/ff_container-landing-section/ff_container-landing-section.less [hunk 2]
```diff
@@ -24,11 +24,7 @@
         text-align: right;
     }
     &__more-link {
-        #ff_mix-util-prose > .link-no-decoration();
-        color: #FFF;
-        &:hover {
-            color: #FFF;
-        }
+        #ff_mix-util-prose > .link-no-decoration(#FFF);
```
Comment (roobottom) [inline blocks/melody/ff_container/ff_container-landing-section/ff_container-landing-section.less:27]: @boonier I know you originally did this, but should this be based off the section colour. like `contrast(@--section--colour)` ? http://lesscss.org/functions/#color-operations-contrast What do you think @dkhuntrods ?

**Comment by dkhuntrods** [comment]
Replaced by https://github.com/fireflylearning/pattern-library/pull/139

**Pull request closed without merging**